README.md

```diff
@@ -65,7 +65,7 @@ For an M1 Mac, read this [SO](https://stackoverflow.com/c/expensify/questions/11
 1. If you are having issues with **_Getting Started_**, please reference [React Native's Documentation](https://reactnative.dev/docs/environment-setup)
```

Comment:
We should update the other examples using `ONYXKEYS.ACCOUNT` to `isLoading` instead of `loading`